kev1n1in/AsiaYoTravelMap
# feature/homepage

[WebUrl](https://asiayo-e20a4--kevin-ta7gp02h.web.app/journey/La2UhBsdnzos7JZvHk8t)

## Description

- Updated the login page to improve user experience.
- Redesigned the homepage for better aesthetics and usability.
- Enhanced the Single Journey edit page with improved functionality and UI adjustments.

## Test Plan


1. Login using Google button:

- Ensure that the Google authentication works correctly and the user is redirected appropriately after login.

2. Navigate to a journey:

- Click on a random journey card on the homepage to enter the map page.
- Verify that the selected journey loads correctly on the map.
- 

3. Delete a journey from the sidebar:

- Attempt to delete a journey using the delete option in the right-side menu.
- Confirm that a warning dialog appears before the journey is deleted.
- Verify that the journey is removed after confirming the deletion in the warning dialog.
